src/sync/condvar.rs

```diff
@@ -38,6 +38,22 @@ impl Condvar {
         Ok(guard)
     }
 
+    /// Blocks the current thread until this condition variable receives a
+    /// notification and the given condition returns false.
```

Comment:
Nit, take it or leave it: we _could_ just copy and paste the [entire doc comment from `std::sync::Condvar`](https://doc.rust-lang.org/stable/std/sync/struct.Condvar.html#method.wait_while) here, since the API is identical. But, it doesn't really matter (and it looks like other methods on our mock `Condvar` don't reproduce the entire `std` comment...

Reply:
I thought about that, but yeah. It didn't seem like that would fit in with the existing comments that were there.